### PR TITLE
Decrypt message

### PR DESCRIPTION
**Pull request by logvik**
Removed the unused new dependency.

added getEncryptionPublicKey function. It will allows getting public key for nacl library.

added description for `getEncryptionPublicKey`

### CODE REVIEW

**Comment by logvik** [comment]
we need it for MetaMask/metamask-extension#1190

**Comment by logvik** [comment]
@lazaridiscom, please review

**Comment by danfinlay** [comment]
Oh sorry, these tests don't run until someone with `write` access pushes a branch to this repo. Was to prevent people from crypto mining on our CI suite.

**Comment by danfinlay** [comment]
Published as `eth-simple-keyring@3.5.0`

File: index.js [hunk 1]
```diff
@@ -92,6 +92,15 @@ class SimpleKeyring extends EventEmitter {
     return Promise.resolve(sig)
   }
 
+  // For eth_decryptMessage:
+  decryptMessage (withAccount, encryptedData) {
```
Comment (lazaridiscom) [inline index.js]: @logvik , I just now (a bit late, as merged) realize that `encryptedData` should be the first parameter here (see sigUtil.decrypt). Should be possibly be revised later (i expect that anyways something else will change).

Comment (lazaridiscom) [reply index.js]: Ignore the above, as the other methods have account/addr parameter first, too, so all fine.

**Comment by lazaridiscom** [comment]
@danfinlay , thanks for the merge. There's still a tiny one needed (non-breaking addition) :

https://github.com/MetaMask/KeyringController/pull/47

**Pull request merged**